website/content/docs/k8s/connect/connect-ca-provider.mdx

```diff
@@ -14,173 +14,8 @@ Consul has support for different certificate authority (CA) providers to be used
 Please see [Connect Certificate Management](/docs/connect/ca) for the information on the providers
 we currently support.
 
-Generally, to configure a provider via the Consul Helm chart, you need to follow three steps:
+To configure a provider via the Consul Helm chart, you need to follow three steps:
 
 1. Create a configuration file containing your provider information.
 1. Create a Kubernetes secret containing the configuration file.
 1. Reference the Kubernetes secret in the [`server.extraVolumes`](/docs/k8s/helm#v-server-extravolumes) value in the Helm chart.
-
```

Comment:
Do you think we should leave this whole section here and just make a note that this applies only to Consul k8s 0.37.0 and below?

Reply:
I'm kinda on the fence with this, since we do not have versioned docs :( generally we go with current release workflows in the docs. I think it might be equally confusing to leave it. thoughts? Is there a way we can flag it `deprecated` somehow?